### ISSUE TITLE
problem with continued lines 

### ISSUE THREAD

**Issue by telecasterer** [bkromhout/vscode-tcl#1]
There a few problems with continued lines.

![image](https://user-images.githubusercontent.com/28565621/64703362-8c9be080-d4ac-11e9-9ee1-5b5ec045b950.png)

**Comment by telecasterer** [comment]
It looks like a continued line that doesn't start with a space is handled correctly.

![image](https://user-images.githubusercontent.com/28565621/64708480-f4562980-d4b4-11e9-8225-735b1257ac4c.png)

BTW did you use iro online to generate the tcl.tmLanguage file from your syntax.iro definition? the reason I ask is that it complains about lookback patterns. Is there perhaps a flag to ignore this?


![image](https://user-images.githubusercontent.com/28565621/64708929-b3aae000-d4b5-11e9-816c-713b6cec74f1.png)

**Comment by bkromhout** [comment]
You're right about the line continuations; I don't use those in my environment, so I didn't think to test for them ;)

I'm actually not sure how easy they'll be to fix given that Textmate Grammars operate on a line-by-line basis, so there's not really any good way to indicate that the current "coloring scope" should continue onto the next line. The fact that it doesn't break if the next line doesn't have leading whitespace is interesting though...

I'll give it a quick look today, but if I can't find a quick fix then it'll have to wait until I have some spare bandwidth, which might not be for a while (I made this between some larger projects at work).

I did indeed use the contents of that syntax.iro file to generate it. For me, the look-behind usages get marked as warnings, not errors:
![image](https://user-images.githubusercontent.com/4447359/64713116-577e9680-d48a-11e9-8ea2-986a6a324f62.png)

I did notice that the warnings/errors system appears to be a little bit "sticky"—sometimes it'll decide something was an error, and if there happened to be a warning as well, then even after the error is fixed, it doesn't "downgrade" to a warning. Typically I was able to fix this by cutting the offending regex out, typing a dummy one that was valid (e.g., `(.)`), then pasting the real one back in. YMMV though, and I can't really offer any better advice than "just fiddle with it some", unfortunately.

The only legitimate error I ran into was that the Iro compiler didn't recognize possessive regex quantifiers (e.g., `*+`, `++`, etc.) despite them being supported by Oniguruma regexes. I only needed this once, in the last pattern in the `command` context (the one right above `: include "args";`), hence the comment (description) I added there noting that it needs to be added by hand to the TextMate grammar that gets output.
